pkg/cloudprovider/aws/suite_test.go

```diff
@@ -61,6 +63,11 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	opts = &options.Options{
+		ClusterName:     "test-cluster",
+		ClusterEndpoint: "https://test-cluster",
+	}
+	ctx = opts.Inject(ctx)
```

Comment:
Might be a bit more idiomatic to not have the method on the options object and instead have `options.Inject(ctx, {})`. This is how the restconfig package does it.

Reply:
done